### ISSUE TITLE
Escaping bare attributes content

### ISSUE THREAD

**Issue by elia** [eagletmt/faml#48]
Passing `signed_in_html` as an attribute value directly doesn't properly escape it

```haml
    - signed_in_html  = render('application/header_signed_in')
    .account.js-account(data-signed-in-html=signed_in_html)
```

instead it seems to work when used in an interpolated string:

```haml
    - signed_in_html  = render('application/header_signed_in')
    .account.js-account(data-signed-in-html="#{signed_in_html}")
```

**Comment by eagletmt** [comment]
Fixed in v0.8.1

**Comment by elia** [comment]
Thanks!
